### ISSUE TITLE
Error from nvim-treesitter when saving a file

### ISSUE THREAD

**Issue by techapostle** [ecosse3/nvim#40]
When saving a file, there is an error thrown that is caused by a deprecated function call from nvim-treesitter.  
The error states:  
``` nvim-treesitter.ts_utils.get_node_text is deprecated: use vim.treesitter.query.get_node_text ```

**Comment by ecosse3** [comment]
Yes, I saw it already too. I think it's some new issue on nvim-treesitter/ts_utils library.

**Comment by ecosse3** [comment]
It's not related to my config but another packages I use in Ecovim. At least nvim-cmp, nvim-gps, hlargs and nvim-autopairs are affected by latest changes to nvim-treesitter. We need to wait for their changes.

**Comment by ecosse3** [comment]
Already fixed 😄
